setup.py

```diff
@@ -56,6 +56,11 @@ def read(*rnames):
     ],
     license='ZPL 2.1',
     url='https://github.com/zopefoundation/zope.meta',
+    project_urls={
+        'Documentation': 'https://zopemeta.readthedocs.io',
+        'Issue Tracker': 'https://github.com/zopefoundation/meta/issues',
+        'Sources': 'https://github.com/zopefoundation/meta',
```

Comment:
This is inconsistent with all the other places that already refer to https://github.com/zopefoundation/zope.meta

Reply:
That's all fixed.